sds_system/accounts/forms.py

```diff
@@ -0,0 +1,23 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.models import User
+
+
+class UserCreateForm(UserCreationForm):
+    email = forms.EmailField(required=True, label='Email', error_messages={'exists': 'This already exists!'})
```

Comment:
instead of defining the form for email validation. Write custom User class and override the email field to be unique and required.